tardis/opacities/opacity_state.py

```diff
@@ -53,7 +57,40 @@ def from_legacy_plasma(cls, plasma, tau_sobolev):
         -------
         OpacityStatePython
         """
+        if hasattr(plasma, "photo_ion_cross_sections"):
+            continuum_state = ContinuumState.from_legacy_plasma(plasma)
+        else:
+            continuum_state = None
+
+        atomic_data = plasma.atomic_data
```

Comment:
Same as before but not sure what declaring this atomic_data variable helps with.